src/bootstrap/bin/main.rs

```diff
@@ -12,5 +12,31 @@ use bootstrap::{Build, Config};
 fn main() {
     let args = env::args().skip(1).collect::<Vec<_>>();
     let config = Config::parse(&args);
+    check_version(&config);
     Build::new(config).build();
 }
+
+fn check_version(config: &Config) {
+    const VERSION: usize = 0;
+
+    let version_suggestion = if let Some(version) = config.version {
+        if version != VERSION {
+            println!(
+                "warning: there have been changes to x.py since you last updated `version = {}`",
```

Comment:
```suggestion
                "warning: there have been changes to x.py since you last updated.",
```

Reply:
I don't think it's all that helpful to tell them the current version in config.toml, or at least, I wasn't able to find a nice way to fit it in here without being very verbose or misleading. The current PR seems too "brief" -- it looks weird to just have a version = {} and is unclear to me which version that is.

Reply:
That makes sense, we're telling them to update `version = VERSION` a few lines later anyway so they know the version and this message are related.